.gitignore

```diff
@@ -72,6 +75,7 @@ python
 Downloads
 src-extern
 notes
+.pylintrc
```

Comment:
It's a python lint configuration settings file. I can actually move this down so it's out of the project entirely and can be used / configured on an individual basis.